ArduPlane/quadplane.cpp

```diff
@@ -735,6 +742,15 @@ bool QuadPlane::setup(void)
     if (!motors->initialised_ok()) {
         AP_BoardConfig::config_error("unknown Q_FRAME_TYPE %u", (unsigned)frame_type.get());
     }
+
+    if (motors_var_info == AP_MotorsMatrix::var_info &&
+        tilt.tilt_mask != 0 &&
+        tilt.tilt_type == TILT_TYPE_VECTORED_YAW) {
+        // we will be using vectoring for yaw
+        motors->disable_yaw_torque();
```

Comment:
We have a couple of no yaw torque frames setup for tailsitters, I wonder if we should either switch to those frame types for this or use this new method for them, I guess that is better since we then support all existing frames with no yaw torque.

https://github.com/ArduPilot/ardupilot/blob/4b47ddf556dd40b03448e7d456bd766c4f1d6613/libraries/AP_Motors/AP_Motors_Class.h#L63

Reply:
yes, calling that fn when we want no yaw torque seems like a good idea